### PR TITLE
Create yamls for attorney generals

### PR DESCRIPTION
**Pull request by OdeclasV**

### CODE REVIEW

File: members/SCOA124_280819.yaml [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          selector: input#edit-submitted-your-contact-information-verify-email
+          value: $EMAIL
+          required: true
+        - name: Message
+          selector: textarea#edit-submitted-your-inquiry-information-please-provide-a-brief-summary-of-your-inquiry
+          value: $MESSAGE
+          required: true
+    - select:
+        - name: State
+          selector: select#edit-submitted-your-contact-information-state
```
Comment (tirish) [inline members/SCOA124_280819.yaml]: Minor: The tag portion of these selectors is unnecessary, as the id alone should be enough. No need to change, just a flag for the future.

File: members/SCOA124_280819.yaml [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          value: Indiana
+          required: true
+    - click_on:
+        - value: Send Email
+          selector: button[type='submit'][value='Submit']
+    - wait:
+        - value: 3
+  success:
+    headers:
```
Comment (tirish) [inline members/SCOA124_280819.yaml]: Minor: just so you know, these `headers` aren't used, so feel free to exclude going forward

File: members/SINA164_589956.yaml [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+          required: true
+        - name: State
+          selector: select#field75702273-state
+          value: Indiana
+          required: true
+    - click_on:
+        - value: Send Email
+          selector: input#fsSubmitButton3396309
+    - wait:
+        - value: 3
```
Comment (tirish) [inline members/SINA164_589956.yaml]: Is it possible to use a `find` instead? That will wait a maximum of 5 seconds but will take less time if it finds it sooner

File: members/SFLA134_627808.yaml [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+            Volusia: Volusia
+            Wakulla: Wakulla
+            Walton: Walton
+            Washington: Washington
+          required: true
+    - click_on:
+        - value: Submit
+          selector: input[type='button'][value='Submit']
+    - wait:
+       - value: 3
```
Comment (tirish) [inline members/SFLA134_627808.yaml]: Is it possible to use a `find` instead? That will wait a maximum of 5 seconds but will take less time if it finds it sooner

File: members/SWIA357_369417.yaml [hunk 5]
```diff
@@ -0,0 +1,53 @@
...
+          required: true
+        - name: Message
+          selector: textarea#edit-submitted-contact-type-your-message
+          value: $MESSAGE
+          required: true
+    - click_on:
+        - value: Submit
+          selector: input[type='submit'][value='Submit']
+    - wait:
+        - value: 3
```
Comment (tirish) [inline members/SWIA357_369417.yaml]: Is it possible to use a `find` instead? That will wait a maximum of 5 seconds but will take less time if it finds it sooner

File: members/SWYA766_702158.yaml [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+    - select:
+        - name: Division
+          selector: select#entry_983581508
+          value: Attorney General
+          required: true
+    - click_on:
+        - value: Submit
+          selector: input[type='submit'][value='Submit']
+    - wait:
+        - value: 3
```
Comment (tirish) [inline members/SWYA766_702158.yaml]: Is it possible to use a `find` instead? That will wait a maximum of 5 seconds but will take less time if it finds it sooner

File: members/SCOL39_250362.yaml [hunk 7]
```diff
@@ -0,0 +1,116 @@
...
+  method: post
+  action: ""
+  steps:
+    - visit: "http://www.tfaforms.com/426053"
+    - find:
+        - selector: input#tfa_2745
+    - click_on:
+        - selector: input#tfa_2745
+    - find:
+        - selector: form#tfa_0 input#tfa_2748
```
Comment (tirish) [inline members/SCOL39_250362.yaml]: Do you need the form id in these selectors? Seems like the `#tfa_2748` alone should be enough

**Comment by OdeclasV** [comment]
@tirish added the `find` steps in the yamls in question (btw, good to know about the 5 second max wait for `find`). Let me know if there are other edit/comments

**Pull request merged**